core/environment.go

```diff
@@ -28,6 +27,8 @@ type (
 		MainFile      *Var
 		args          *Var
 		classPath     *Var
+		verbose       *Var
```

Comment:
This var is not used anymore and can be deleted, right?

Reply:
No, and neither is `verbosity` needed, nor the code that initializes them.

Gone!